resources/charlie/validator_keys/README.md

```diff
@@ -0,0 +1,19 @@
+## Validator Keys  
+
+Keys should be created by running (assuming you are in this directory for local path "."):
```

Comment:
I expect we won't clear out this folder when upgrading or uninstalling, so should the docs here mention that keys only need creating if they don't exist already?